orchestra/static/orchestra/dashboard/tasktable.scss

```diff
@@ -1,11 +1,3 @@
 .collapsed-toggle {
   cursor: pointer;
 }
-
-.tags-col {
```

Comment:
Weirdly, I see we removed it, but i don't see where we moved it to. But I also see it in the compiled .css files. Am I being silly?

Reply:
I moved it to `orchestra/static/dist/orchestra/common/css/orchestra.css`. You are not being silly because `orchestra/static/dist/orchestra/common/css/orchestra.css` is not compiled css. :)

Reply:
i could be wrong about this, but I believe that
* https://github.com/b12io/orchestra/blob/main/orchestra/static/dist/orchestra/common/css/orchestra.css comes from
* https://github.com/b12io/orchestra/blob/main/orchestra/static/orchestra/common/css/orchestra.scss, which includes
* https://github.com/b12io/orchestra/blob/main/orchestra/static/communication/scss/_communication.scss, which is has the CSS for available tasks.

and importantly, editing orchestra.css directly will make it get overwritten the next time we rebuilt it. I *think* the CSS gets compiled via `gulp` here: https://github.com/b12io/orchestra/blob/main/gulpfile.js#L35

Reply:
You are right! Thanks for catching that! It was being compiled with `gulp`. However, on compiling with gulp, it adds this extra line of `/*# sourceMappingURL=data:application/json;charset=utf8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoib3JjaGVzdHJhL3N0YXRpYy9vcmNoZXN0cmEvY29tbW9uL2Nzcy9vcmNoZXN0cmEuY3NzIiwic291cmNlcyI6WyJvcmNoZXN0cmEvc3RhdGljL29yY2hlc3RyYS9jb21tb24vY3NzL29yY2hlc3RyYS5zY3NzIiwib3JjaGVzdHJhL3N0YXRpYy9hY2NvdW50cy9zY3NzL19yZWdpc3RyYXRpb24uc2NzcyIsIm9yY2hlc3RyYS9zdGF0aWMvb3JjaGVzdHJhL2NvbW1vbi9jc3MvX2NvbW1vbi5zY3NzIiwib3JjaGVzdHJhL3N0YXRpYy9vcmNoZXN0cmEvY29tbW9uL2Nzcy9fYmFzZS5zY3NzIiwib3JjaGVzdHJhL3N0YXRpYy9vcmNoZXN0cmEvY29tbW9uL2Nzcy9fZXJyb3Iuc2NzcyIsIm9yY2hlc3RyYS9zdGF0aWMvYWNjb3VudHMvc2Nzcy9fc2V0dGluZ3Muc2NzcyIsIm9yY2hlc3RyYS9zdGF0aWMvY29tbXVuaWNhdGlvbi9zY3NzL19jb21tdW5pY2F0aW9uLnNjc3MiXSwic291cmNlc0NvbnRlbnQiOlsiQGltcG9ydCAnb3JjaGVzdHJhL3N0YXRpYy9hY2NvdW50cy9zY3NzL3JlZ2lzdHJhdGlvbic7XG5AaW1wb3J0ICdvcmNoZXN0cmEvc3RhdGljL29yY2hlc3RyYS9jb21tb24vY3NzL2Vycm9yJztcbkBpbXBvcnQgJ29yY2hlc3RyYS9zdGF0aWMvYWNjb3VudHMvc2Nzcy9zZXR0aW5ncyc7XG5AaW1wb3J0ICdvcmNoZXN0cmEvc3RhdGljL2NvbW11bmljYXRpb24vc2Nzcy9jb21tdW5pY2F0aW9uJztcbiIsIkBpbXBvcnQgJ29yY2hlc3RyYS9zdGF0aWMvb3JjaGVzdHJhL2NvbW1vbi9jc3MvY29tbW9uJztcbkBpbXBvcnQgJ29yY2hlc3RyYS9zdGF0aWMvb3JjaGVzdHJhL2NvbW1vbi9jc3MvYmFzZSc7XG5cbi8qIExvZ2luL2FjY291bnQtcmVsYXRlZCBwYWdlcyAqL1xuLmxvZ2luLXdyYXAge1xuICBmb250LXNpemU6IDEuM3JlbTtcbiAgLmVycm9yLW5vbi1maWVsZCB7XG4gICAgcGFkZGluZzogMHB4IDBweCAxMHB4IDEwcHg7XG4gICAgY29sb3I6IHJlZDtcbiAgfVxuICAuZXJyb3ItZmllbGQge1xuICAgIHBhZGRpbmc6IDVweCAwcHggMHB4IDEwcHg7XG4gICAgY29sb3I6IHJlZDtcbiAgfVxufVxuIiwiLyoqXG4gKiBDb21tb24gc3R5bGVzaGVldCBmb3IgT3JjaGVzdHJhLlxuICpcbiAqIFRoaXMgcGFydGlhbCBwcm92aWRlcyBjb21tb24gdmFyaWFibGVzIGFuZCBtaXhpbnMgdXNlZnVsIGZvciB2YXJpb3VzIHZpZXcgYW5kXG4gKiBjb21wb25lbnQgc3R5bGVzLlxuICovXG5cbiRtYWluLWJsdWU6IHJnYigwLCAxMjEsIDE5MSk7XG4kbGlnaHQtZ3JheTogcmdiKDIwNCwgMjA0LCAyMDQpO1xuJGRhcmstZ3JheTogcmdiKDYwLCA2MCwgNjApO1xuJGxpZ2h0LWdyZWVuOiByZ2IoNDYsIDIwNCwgMTEzKTtcbiRkYXJrLWdyZWVuOiByZ2IoMzksIDE3NCwgOTYpO1xuJGJvbGQteWVsbG93OiByZ2IoMjMwLCAxOTgsIDU1KTtcbiRibGFjazogcmdiKDAsIDAsIDApO1xuXG4kc3FydC0yOiAxLjQxNDI7XG5cbiRiYXNlLWZvbnQtc2l6ZTogMTBweDtcbiRwYW5lbC1tYXJnaW46IDNyZW07XG4kcGFuZWwtcGFkZGluZzogMS41cmVtO1xuJHBhbmVsLWJ1dHRvbi1tYXJnaW46IDEuNXJlbTtcblxuJG1haW4tZm9udC1zdGFjazogLWFwcGxlLXN5c3RlbSwgQmxpbmtNYWNTeXN0ZW1Gb250LCBcIlNlZ29lIFVJXCIsIFJvYm90bywgSGVsdmV0aWNhLCBBcmlhbCwgc2Fucy1zZXJpZiwgXCJBcHBsZSBDb2xvciBFbW9qaVwiLCBcIlNlZ29lIFVJIEVtb2ppXCIsIFwiU2Vnb2UgVUkgU3ltYm9sXCI7XG5cbkBtaXhpbiB2ZW5kb3ItcHJlZml4KCRuYW1lLCAkYXJndW1lbnQpIHtcbiAgLXdlYmtpdC0jeyRuYW1lfTogI3skYXJndW1lbnR9O1xuICAtbXMtI3skbmFtZX06ICN7JGFyZ3VtZW50fTtcbiAgLW1vei0jeyRuYW1lfTogI3skYXJndW1lbnR9O1xuICAtby0jeyRuYW1lfTogI3skYXJndW1lbnR9O1xuICAjeyRuYW1lfTogI3skYXJndW1lbnR9O1xufVxuXG4uZGlzYWJsZS1zZWxlY3Qge1xuICAtd2Via2l0LXRvdWNoLWNhbGxvdXQ6IG5vbmU7XG4gIC13ZWJraXQtdXNlci1zZWxlY3Q6IG5vbmU7XG4gIC1raHRtbC11c2VyLXNlbGVjdDogbm9uZTtcbiAgLW1vei11c2VyLXNlbGVjdDogbm9uZTtcbiAgLW1zLXVzZXItc2VsZWN0OiBub25lO1xuICB1c2VyLXNlbGVjdDogbm9uZTtcbn1cblxuJW5vLXBhZGRpbmcge1xuICBwYWRkaW5nLWxlZnQ6IDA7XG4gIHBhZGRpbmctcmlnaHQ6IDA7XG59XG5cbi5yb3cubm8tcGFkZGluZyB7XG4gIEBleHRlbmQgJW5vLXBhZGRpbmc7XG4gID4gW2NsYXNzICo9IFwiY29sLVwiXSB7XG4gICAgQGV4dGVuZCAlbm8tcGFkZGluZztcbiAgfVxufVxuXG4lbG9hZGluZy1zcGlubmVyIHtcbiAgJjo6YWZ0ZXIge1xuICAgIHBvc2l0aW9uOiBhYnNvbHV0ZTtcbiAgICB0b3A6IDA7XG4gICAgbGVmdDogMDtcbiAgICByaWdodDogMDtcbiAgICBib3R0b206IDA7XG4gICAgYm9yZGVyLWJvdHRvbTogMXB4IHNvbGlkICRibGFjaztcbiAgICBib3JkZXItcmlnaHQ6IDFweCBzb2xpZCAkYmxhY2s7XG4gICAgYm9yZGVyLXJhZGl1czogNTAlO1xuICAgIGNvbnRlbnQ6IFwiXCI7XG4gICAgYW5pbWF0aW9uOiBzcGluIC43NXMgbGluZWFyIGluZmluaXRlO1xuICB9XG59XG5cbkBrZXlmcmFtZXMgc3BpbiB7XG4gIGZyb20ge1xuICAgIHRyYW5zZm9ybTpyb3RhdGUoMGRlZyk7XG4gIH1cbiAgdG8ge1xuICAgICB0cmFuc2Zvcm06cm90YXRlKDM2MGRlZyk7XG4gIH1cbn1cbiIsIi8qKlxuICogQmFzZSBzdHlsZXNoZWV0IGZvciBPcmNoZXN0cmEuXG4gKlxuICogVGhpcyBwYXJ0aWFsIHN0eWxlcyB0aGUgYmFzZSBEamFuZ28gdGVtcGxhdGUgYW5kIHNlcnZlcyBhcyB0aGUgZm91bmRhdGlvbiBmb3JcbiAqIGVhY2ggT3JjaGVzdHJhIHZpZXcuXG4gKi9cblxuJGhlYWRlci1oZWlnaHQ6IDUycHg7XG5cbmh0bWwsIGJvZHkge1xuICBmb250LXNpemU6ICRiYXNlLWZvbnQtc2l6ZTtcbn1cblxuc2VjdGlvbi53cmFwcGVyIHtcbiAgbWFyZ2luLXRvcDogJGhlYWRlci1oZWlnaHQ7XG59XG5cbnNlY3Rpb24jbWFpbi1jb250ZW50IHtcbiAgbWFyZ2luLWxlZnQ6IDA7XG4gIGNvbG9yOiAkZGFyay1ncmF5O1xufVxuXG5oZWFkZXIuaGVhZGVyIHtcbiAgYmFja2dyb3VuZDogZGFya2VuKCRtYWluLWJsdWUsIDEwKTtcbiAgYm94LXNoYWRvdzogMCAxcHggM3B4IGJsYWNrO1xuICBtaW4taGVpZ2h0OiAwO1xufVxuXG5oMSwgaDIsIGgzLCBoNCwgaDUsIGg2LCBwIHtcbiAgbWFyZ2luOiAwO1xuICBwYWRkaW5nOiAwXG59XG5cbmEge1xuICBjb2xvcjogJG1haW4tYmx1ZTtcbiAgdGV4dC1kZWNvcmF0aW9uOiBub25lO1xuICAmOmhvdmVyLCAmOmZvY3VzLCAmOnZpc2l0ZWQge1xuICAgIGNvbG9yOiAkbWFpbi1ibHVlO1xuICAgIHRleHQtZGVjb3JhdGlvbjogdW5kZXJsaW5lO1xuICB9XG59XG5cbmEubG9nbyB7XG4gIGZvbnQtc2l6ZTogMnJlbTtcbiAgY29sb3I6IHdoaXRlO1xuICBtYXJnaW46IDEycHggMzBweDtcbiAgJjpob3ZlciwgJjpmb2N1cywgJjp2aXNpdGVkIHtcbiAgICBjb2xvcjogd2hpdGU7XG4gIH1cbn1cblxuLndvcmstdGltZXIge1xuICBmbG9hdDogcmlnaHQ7XG4gIGhlaWdodDogJGhlYWRlci1oZWlnaHQ7XG59XG5cbiNuYXYtb3B0aW9ucyB7XG4gIGN1cnNvcjogcG9pbnRlcjtcbiAgZm9udC1zaXplOiAycmVtO1xuICBjb2xvcjogd2hpdGU7XG4gIGZsb2F0OiByaWdodDtcbiAgbWFyZ2luOiAxMnB4IDMwcHg7XG4gIHotaW5kZXg6IDEwMDM7XG4gIC5kcm9wZG93bi1tZW51IHtcbiAgICBhIHtcbiAgICAgIGNvbG9yOiAkbWFpbi1ibHVlO1xuICAgICAgJjpob3ZlciB7XG4gICAgICAgIGNvbG9yOiAkbWFpbi1ibHVlO1xuICAgICAgICB0ZXh0LWRlY29yYXRpb246IHVuZGVybGluZTtcbiAgICAgIH1cbiAgICB9XG4gIH1cbn1cblxuI2hpamFja2VkLXdhcm5pbmcge1xuICBsZWZ0OiAyNSU7XG4gIHdpZHRoOiA1MCU7XG4gIGJvcmRlci1yYWRpdXM6IDAgMCA1cHggNXB4O1xuICA+IHNwYW46bGFzdC1jaGlsZCB7XG4gICAgLy8gSGFjayB0byBvdmVycmlkZSBidXR0b24gY29udGFpbmVyIGlubGluZSBzdHlsZVxuICAgIG1hcmdpbi1yaWdodDogMCAhaW1wb3J0YW50O1xuICB9XG59XG5cbi5zZWN0aW9uLXBhbmVsIHtcbiAgbWFyZ2luOiAkcGFuZWwtYnV0dG9uLW1hcmdpbjtcbiAgYm94LXNoYWRvdzogMCAxcHggM3B4IGJsYWNrO1xuICBiYWNrZ3JvdW5kLWNvbG9yOiB3aGl0ZTtcbiAgLnNlY3Rpb24taGVhZGVyIHtcbiAgICBtYXJnaW4tdG9wOiAwO1xuICAgIGNvbG9yOiB3aGl0ZTtcbiAgICBiYWNrZ3JvdW5kLWNvbG9yOiAkbWFpbi1ibHVlO1xuICAgIGgzIHtcbiAgICAgIG1hcmdpbjogMS4zcmVtIDA7XG4gICAgICBmb250LXNpemU6IDEuOHJlbTtcbiAgICB9XG4gICAgLmJ0biB7XG4gICAgICBkaXNwbGF5OiBpbmxpbmUtYmxvY2s7XG4gICAgICBjb2xvcjogd2hpdGU7XG4gICAgICBiYWNrZ3JvdW5kLWNvbG9yOiAkbGlnaHQtZ3JlZW47XG4gICAgICAmOmZpcnN0LWNoaWxkIHtcbiAgICAgICAgbWFyZ2luLWxlZnQ6ICRwYW5lbC1idXR0b24tbWFyZ2luO1xuICAgICAgfVxuICAgICAgJjpob3Zlcjpub3QoLmRpc2FibGVkKSB7XG4gICAgICAgIGJvcmRlcjogMXB4IHNvbGlkIHdoaXRlO1xuICAgICAgfVxuICAgICAgJi5kaXNhYmxlZCB7XG4gICAgICAgIGNvbG9yOiAkbGlnaHQtZ3JheTtcbiAgICAgICAgYmFja2dyb3VuZC1jb2xvcjogcmdiYSgkbGlnaHQtZ3JlZW4sIDAuNik7XG4gICAgICB9XG4gICAgfVxuICB9XG4gIC5yb3c6bm90KC5zZWN0aW9uLWhlYWRlcikge1xuICAgICYucGFkZGVkIHtcbiAgICAgID4gW2NsYXNzKj1cImNvbC1cIl0ge1xuICAgICAgICBwYWRkaW5nLXRvcDogJHBhbmVsLXBhZGRpbmc7XG4gICAgICAgIHBhZGRpbmctYm90dG9tOiAkcGFuZWwtcGFkZGluZztcbiAgICAgIH1cbiAgICB9XG4gIH1cbn1cblxuLm1haW4tY29udGVudCB7XG4gIG1hcmdpbjogMTAwcHggYXV0byAwO1xufVxuIiwiLmVycm9yLXBhZ2Uge1xuICAgIHRleHQtYWxpZ246IGNlbnRlcjtcbiAgICBsaW5lLWhlaWdodDogMjRweDtcbiAgICBjb2xvcjogIzAwMDAwMDtcbn1cblxuLmVycm9yLWRpc3BsYXkge1xuICAgIGZvbnQtc2l6ZTogNnJlbTtcbiAgICBtYXJnaW4tYm90dG9tOiAxNXB4O1xufVxuXG4uZXJyb3ItdGV4dCB7XG4gICAgZm9udC1zaXplOiAycmVtO1xuICAgIG1hcmdpbi1ib3R0b206IC41cmVtO1xufVxuIiwiLmFjY291bnQtc2V0dGluZ3Mge1xuXG4gIC5hbGVydCB7XG4gICAgbWFyZ2luLXRvcDogLTcwcHg7XG4gIH1cblxuICBhOmhvdmVyLCBhOmZvY3VzLCBhOnZpc2l0ZWQge1xuICAgIGNvbG9yOiAjRkZGO1xuICAgIHRleHQtZGVjb3JhdGlvbjogbm9uZTtcbiAgfVxuXG4gIC5yb3c6bm90KDpsYXN0LWNoaWxkKSB7XG4gICAgbWFyZ2luLWJvdHRvbTogMTVweDtcbiAgfVxuXG4gIC5hY2NvdW50LXRhYnMge1xuICAgIHBvc2l0aW9uOiByZWxhdGl2ZTtcbiAgICB6LWluZGV4OiAzO1xuICAgIC5idG4ge1xuICAgICAgYm9yZGVyLWJvdHRvbS1sZWZ0LXJhZGl1czogMDtcbiAgICAgIGJvcmRlci1ib3R0b20tcmlnaHQtcmFkaXVzOiAwO1xuICAgICAgJi5zZWxlY3RlZCwgJjpob3ZlcntcbiAgICAgICAgY29sb3I6IGRhcmtlbigjNDI4YmNhLCAxNSUpO1xuICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiB3aGl0ZTtcbiAgICAgICAgYm9yZGVyLWNvbG9yOiB3aGl0ZTtcbiAgICAgIH1cbiAgICB9XG4gIH1cblxuICAuY29tbXVuaWNhdGlvbi1tZXRob2RzIHtcbiAgICB1bCB7XG4gICAgICBwYWRkaW5nOiA1cHggMHB4IDBweCAwcHg7XG4gICAgfVxuICAgIHVsIHAgbGFiZWwge1xuICAgICAgZm9udC1zaXplOiAxM3B4O1xuICAgICAgZm9udC13ZWlnaHQ6IG5vcm1hbDtcbiAgICAgIGxldHRlci1zcGFjaW5nOiBub3JtYWw7XG4gICAgICBsaW5lLWhlaWdodDogaW5oZXJpdDtcbiAgICB9XG4gIH1cblxuICAuY2FyZCB7XG4gICAgcG9zaXRpb246IHJlbGF0aXZlO1xuICAgIHotaW5kZXg6IDI7XG4gICAgcGFkZGluZzogNTBweDtcbiAgICBiYWNrZ3JvdW5kOiAjRkZGRkZGO1xuICAgIGJveC1zaGFkb3c6IDBweCAxN3B4IDI1cHggMHB4IHJnYmEoMTA0LCAxMDQsIDEwNCwgMC4wNCksIDBweCAwcHggNTVweCAwcHggcmdiYSgyMTksIDIxOSwgMjE5LCAwLjI4KTtcbiAgICBoZWlnaHQ6IGF1dG87XG4gIH1cblxuICAuYWNjb3VudC1hY3Rpb25zIHtcbiAgICBtYXJnaW4tdG9wOiAxNXB4O1xuICAgIHRleHQtYWxpZ246IHJpZ2h0O1xuICB9XG5cbiAgbGFiZWwge1xuICAgIGZvbnQtc2l6ZTogMTRweDtcbiAgICBmb250LXdlaWdodDogNjAwO1xuICAgIGxldHRlci1zcGFjaW5nOiAxLjU5cHg7XG4gICAgbGluZS1oZWlnaHQ6IDI2cHg7XG4gIH1cblxuICBpbnB1dFt0eXBlPWNoZWNrYm94XSB7XG4gICAgaGVpZ2h0OiBpbmhlcml0O1xuICAgIHdpZHRoOiBhdXRvO1xuICAgIGRpc3BsYXk6IGlubGluZS1ibG9jaztcbiAgICBtYXJnaW4tbGVmdDogM3B4O1xuICAgIG1hcmdpbi1yaWdodDogNXB4O1xuICB9XG5cbiAgI2lkX3Bob25lXzAsICNpZF9waG9uZV8xIHtcbiAgICBkaXNwbGF5OiBpbmxpbmU7XG4gIH1cblxuICAjaWRfcGhvbmVfMCB7XG4gICAgd2lkdGg6IDI5JTtcbiAgICBtYXJnaW4tcmlnaHQ6IDElO1xuICB9XG5cbiAgI2lkX3Bob25lXzEge1xuICAgIHdpZHRoOiA3MCU7XG4gIH1cblxuICAuYXZhaWxhYmlsaXR5LWZvcm1zIHtcbiAgICBkaXNwbGF5OiBmbGV4O1xuICAgIGp1c3RpZnktY29udGVudDogc3BhY2UtYmV0d2VlbjtcbiAgfVxuXG4gIC5hdmFpbGFiaWxpdHktZm9ybSB7XG4gICAgaW5wdXQge1xuICAgICAgbWF4LXdpZHRoOiAxMDBweDtcbiAgICB9XG5cbiAgICB0ZDpmaXJzdC1jaGlsZCB7XG4gICAgICB0ZXh0LWFsaWduOiByaWdodDtcbiAgICAgIHBhZGRpbmc6IDAgMTJweCA2cHggMDtcbiAgICB9XG5cbiAgICBsYWJlbCB7XG4gICAgICBtYXJnaW4tYm90dG9tOiAwO1xuICAgICAgZm9udC1zaXplOiAxM3B4O1xuICAgICAgZm9udC13ZWlnaHQ6IDQwMDtcbiAgICB9XG5cbiAgICAmX19oZWFkaW5nIHtcbiAgICAgIGZvbnQtc2l6ZTogMTRweDtcbiAgICAgIGZvbnQtd2VpZ2h0OiA2MDA7XG4gICAgfVxuICB9XG59XG4iLCIuY29tbXVuaWNhdGlvbi1jb250YWluZXIge1xuICB0ZXh0LWFsaWduOiBjZW50ZXI7XG5cbiAgLmNhcmQge1xuICAgIHBhZGRpbmctdG9wOiA4MHB4O1xuICB9XG5cbiAgYSB7XG4gICAgY29sb3I6ICNGRkY7XG4gIH1cblxuICAuc3BhY2VkIHtcbiAgICBwYWRkaW5nLXRvcDogMTVweDtcbiAgfVxufVxuXG5cbi5zdGFmZmluZy1jb250YWluZXIge1xuICBmb250LXNpemU6IDEuMmVtO1xuICAuY2FyZCB7XG4gICAgaGVpZ2h0OiBpbml0aWFsO1xuXG4gICAgaDIge1xuICAgICAgcGFkZGluZzogMTI1cHggMHB4IDEyNXB4IDBweDtcbiAgICAgIHRleHQtYWxpZ246IGNlbnRlcjtcbiAgICB9XG5cbiAgICB0YWJsZSB7XG4gICAgICB0YWJsZS1sYXlvdXQ6IGZpeGVkO1xuXG4gICAgICB0ZDpmaXJzdC1jaGlsZCwgdGg6Zmlyc3QtY2hpbGQge1xuICAgICAgICBwYWRkaW5nLWxlZnQ6IDIwcHg7XG4gICAgICB9XG5cbiAgICAgIC5yb3ctcHJvamVjdCB7XG4gICAgICAgIHdpZHRoOiAxNSU7XG4gICAgICB9XG4gICAgICAucm93LXByb2plY3QtZGVzY3JpcHRpb24ge1xuICAgICAgICB3aWR0aDogMjAlO1xuICAgICAgfVxuICAgICAgLnJvdy10YXNrIHtcbiAgICAgICAgd2lkdGg6IDIwJTtcbiAgICAgIH1cbiAgICAgIC5yb3ctZGV0YWlscyB7XG4gICAgICAgIHdpZHRoOiAyMCU7XG4gICAgICB9XG4gICAgICAucm93LWNyZWF0ZWQtYXQge1xuICAgICAgICB3aWR0aDogMTAlO1xuICAgICAgfVxuXG4gICAgICAucm93LWFjdGlvbnMge1xuICAgICAgICB3aWR0aDogMTUlO1xuICAgICAgfVxuICAgIH1cbiAgfVxufVxuXG4udGFncy1jb2wge1xuICBtYXgtd2lkdGg6IDE1MHB4O1xufVxuXG4udGFncy1jb2wgc3BhbiB7XG4gIGRpc3BsYXk6IGlubGluZS1ibG9jaztcbiAgbWFyZ2luLXJpZ2h0OiAzcHg7XG59XG4iXSwibWFwcGluZ3MiOiJBRUFBOzs7OztHQUtHO0FBMkJILEFBQUEsZUFBZSxDQUFDO0VBQ2QscUJBQXFCLEVBQUUsSUFBSztFQUM1QixtQkFBbUIsRUFBRSxJQUFLO0VBQzFCLGtCQUFrQixFQUFFLElBQUs7RUFDekIsZ0JBQWdCLEVBQUUsSUFBSztFQUN2QixlQUFlLEVBQUUsSUFBSztFQUN0QixXQUFXLEVBQUUsSUFBSyxHQUNuQjs7QUFPRCxBQUxBLElBS0ksQUFBQSxXQUFXLEVBQWYsQUFMQSxJQUtJLEFBQUEsV0FBVyxJQUVYLEFBQUEsS0FBQyxFQUFTLE1BQU0sQUFBZixFQVBPO0VBQ1YsWUFBWSxFQUFFLENBQUU7RUFDaEIsYUFBYSxFQUFFLENBQUUsR0FDbEI7O0FBd0JELFVBQVUsQ0FBVixJQUFVO0VBQ1IsQUFBQSxJQUFJO0lBQ0YsU0FBUyxFQUFDLFlBQU07RUFFbEIsQUFBQSxFQUFFO0lBQ0MsU0FBUyxFQUFDLGNBQU07O0FDekVyQjs7Ozs7R0FLRztBQUlILEFBQUEsSUFBSSxFQUFFLEFBQUEsSUFBSSxDQUFDO0VBQ1QsU0FBUyxFRE9NLElBQUksR0NOcEI7O0FBRUQsQUFBTyxPQUFBLEFBQUEsUUFBUSxDQUFDO0VBQ2QsVUFBVSxFQVBJLElBQUksR0FRbkI7O0FBRUQsQUFBTyxPQUFBLEFBQUEsYUFBYSxDQUFDO0VBQ25CLFdBQVcsRUFBRSxDQUFFO0VBQ2YsS0FBSyxFRFZLLE9BQUcsR0NXZDs7QUFFRCxBQUFNLE1BQUEsQUFBQSxPQUFPLENBQUM7RUFDWixVQUFVLEVBQUUsT0FBTTtFQUNsQixVQUFVLEVBQUUsZUFBZ0I7RUFDNUIsVUFBVSxFQUFFLENBQUUsR0FDZjs7QUFFRCxBQUFBLEVBQUUsRUFBRSxBQUFBLEVBQUUsRUFBRSxBQUFBLEVBQUUsRUFBRSxBQUFBLEVBQUUsRUFBRSxBQUFBLEVBQUUsRUFBRSxBQUFBLEVBQUUsRUFBRSxBQUFBLENBQUMsQ0FBQztFQUN4QixNQUFNLEVBQUUsQ0FBRTtFQUNWLE9BQU8sRUFBRSxDQUNWLEdBQUM7O0FBRUYsQUFBQSxDQUFDLENBQUM7RUFDQSxLQUFLLEVEM0JLLE9BQUc7RUM0QmIsZUFBZSxFQUFFLElBQUssR0FLdkI7RUFQRCxBQUFBLENBQUMsQUFHRSxNQUFNLEVBSFQsQUFBQSxDQUFDLEFBR1csTUFBTSxFQUhsQixBQUFBLENBQUMsQUFHb0IsUUFBUSxDQUFDO0lBQzFCLEtBQUssRUQ5QkcsT0FBRztJQytCWCxlQUFlLEVBQUUsU0FBVSxHQUM1Qjs7QUFHSCxBQUFDLENBQUEsQUFBQSxLQUFLLENBQUM7RUFDTCxTQUFTLEVBQUUsSUFBSztFQUNoQixLQUFLLEVBQUUsS0FBTTtFQUNiLE1BQU0sRUFBRSxTQUFVLEdBSW5CO0VBUEQsQUFBQyxDQUFBLEFBQUEsS0FBSyxBQUlILE1BQU0sRUFKVCxBQUFDLENBQUEsQUFBQSxLQUFLLEFBSU0sTUFBTSxFQUpsQixBQUFDLENBQUEsQUFBQSxLQUFLLEFBSWUsUUFBUSxDQUFDO0lBQzFCLEtBQUssRUFBRSxLQUFNLEdBQ2Q7O0FBR0gsQUFBQSxXQUFXLENBQUM7RUFDVixLQUFLLEVBQUUsS0FBTTtFQUNiLE1BQU0sRUE5Q1EsSUFBSSxHQStDbkI7O0FBRUQsQUFBQSxZQUFZLENBQUM7RUFDWCxNQUFNLEVBQUUsT0FBUTtFQUNoQixTQUFTLEVBQUUsSUFBSztFQUNoQixLQUFLLEVBQUUsS0FBTTtFQUNiLEtBQUssRUFBRSxLQUFNO0VBQ2IsTUFBTSxFQUFFLFNBQVU7RUFDbEIsT0FBTyxFQUFFLElBQUssR0FVZjtFQWhCRCxBQVFJLFlBUlEsQ0FPVixjQUFjLENBQ1osQ0FBQyxDQUFDO0lBQ0EsS0FBSyxFRDFEQyxPQUFHLEdDK0RWO0lBZEwsQUFRSSxZQVJRLENBT1YsY0FBYyxDQUNaLENBQUMsQUFFRSxNQUFNLENBQUM7TUFDTixLQUFLLEVENURELE9BQUc7TUM2RFAsZUFBZSxFQUFFLFNBQVUsR0FDNUI7O0FBS1AsQUFBQSxpQkFBaUIsQ0FBQztFQUNoQixJQUFJLEVBQUUsR0FBSTtFQUNWLEtBQUssRUFBRSxHQUFJO0VBQ1gsYUFBYSxFQUFFLFdBQVksR0FLNUI7RUFSRCxBQUlRLGlCQUpTLEdBSWIsSUFBSSxBQUFBLFdBQVcsQ0FBQztJQUVoQixZQUFZLEVBQUUsWUFBYSxHQUM1Qjs7QUFHSCxBQUFBLGNBQWMsQ0FBQztFQUNiLE1BQU0sRURqRWMsTUFBTTtFQ2tFMUIsVUFBVSxFQUFFLGVBQWdCO0VBQzVCLGdCQUFnQixFQUFFLEtBQU0sR0FpQ3pCO0VBcENELEFBSUUsY0FKWSxDQUlaLGVBQWUsQ0FBQztJQUNkLFVBQVUsRUFBRSxDQUFFO0lBQ2QsS0FBSyxFQUFFLEtBQU07SUFDYixnQkFBZ0IsRURwRlIsT0FBRyxHQ3dHWjtJQTNCSCxBQVFJLGNBUlUsQ0FJWixlQUFlLENBSWIsRUFBRSxDQUFDO01BQ0QsTUFBTSxFQUFFLFFBQVM7TUFDakIsU0FBUyxFQUFFLE1BQU8sR0FDbkI7SUFYTCxBQVlJLGNBWlUsQ0FJWixlQUFlLENBUWIsSUFBSSxDQUFDO01BQ0gsT0FBTyxFQUFFLFlBQWE7TUFDdEIsS0FBSyxFQUFFLEtBQU07TUFDYixnQkFBZ0IsRUR6RlIsT0FBRyxHQ29HWjtNQTFCTCxBQVlJLGNBWlUsQ0FJWixlQUFlLENBUWIsSUFBSSxBQUlELFlBQVksQ0FBQztRQUNaLFdBQVcsRURqRkcsTUFBTSxHQ2tGckI7TUFsQlAsQUFZSSxjQVpVLENBSVosZUFBZSxDQVFiLElBQUksQUFPRCxNQUFNLEFBQUEsSUFBSyxDQUFBLEFBQUEsU0FBUyxFQUFFO1FBQ3JCLE1BQU0sRUFBRSxlQUFnQixHQUN6QjtNQXJCUCxBQVlJLGNBWlUsQ0FJWixlQUFlLENBUWIsSUFBSSxBQVVELFNBQVMsQ0FBQztRQUNULEtBQUssRURuR0EsT0FBRztRQ29HUixnQkFBZ0IsRURsR1YsdUJBQUcsR0NtR1Y7RUF6QlAsQUE4QnNCLGNBOUJSLENBNEJaLElBQUksQUFBQSxJQUFLLENBQUEsQUFBQSxlQUFlLENBQ3JCLE9BQU8sSUFDSixBQUFBLEtBQUMsRUFBTyxNQUFNLEFBQWIsRUFBZTtJQUNoQixXQUFXLEVEaEdILE1BQU07SUNpR2QsY0FBYyxFRGpHTixNQUFNLEdDa0dmOztBQUtQLEFBQUEsYUFBYSxDQUFDO0VBQ1osTUFBTSxFQUFFLFlBQWEsR0FDdEI7O0FGekhELGlDQUFpQztBQUNqQyxBQUFBLFdBQVcsQ0FBQztFQUNWLFNBQVMsRUFBRSxNQUFPLEdBU25CO0VBVkQsQUFFRSxXQUZTLENBRVQsZ0JBQWdCLENBQUM7SUFDZixPQUFPLEVBQUUsaUJBQWtCO0lBQzNCLEtBQUssRUFBRSxHQUFJLEdBQ1o7RUFMSCxBQU1FLFdBTlMsQ0FNVCxZQUFZLENBQUM7SUFDWCxPQUFPLEVBQUUsZ0JBQWlCO0lBQzFCLEtBQUssRUFBRSxHQUFJLEdBQ1o7O0FHYkgsQUFBQSxXQUFXLENBQUM7RUFDUixVQUFVLEVBQUUsTUFBTztFQUNuQixXQUFXLEVBQUUsSUFBSztFQUNsQixLQUFLLEVBQUUsT0FBUSxHQUNsQjs7QUFFRCxBQUFBLGNBQWMsQ0FBQztFQUNYLFNBQVMsRUFBRSxJQUFLO0VBQ2hCLGFBQWEsRUFBRSxJQUFLLEdBQ3ZCOztBQUVELEFBQUEsV0FBVyxDQUFDO0VBQ1IsU0FBUyxFQUFFLElBQUs7RUFDaEIsYUFBYSxFQUFFLEtBQU0sR0FDeEI7O0FDZEQsQUFFRSxpQkFGZSxDQUVmLE1BQU0sQ0FBQztFQUNMLFVBQVUsRUFBRSxLQUFNLEdBQ25COztBQUpILEFBTUcsaUJBTmMsQ0FNZixDQUFDLEFBQUEsTUFBTSxFQU5ULEFBTVksaUJBTkssQ0FNTixDQUFDLEFBQUEsTUFBTSxFQU5sQixBQU1xQixpQkFOSixDQU1HLENBQUMsQUFBQSxRQUFRLENBQUM7RUFDMUIsS0FBSyxFQUFFLElBQUs7RUFDWixlQUFlLEVBQUUsSUFBSyxHQUN2Qjs7QUFUSCxBQVdzQixpQkFYTCxDQVdmLElBQUksQUFBQSxJQUFLLENBQUEsQUFBQSxXQUFXLEVBQUU7RUFDcEIsYUFBYSxFQUFFLElBQUssR0FDckI7O0FBYkgsQUFlRSxpQkFmZSxDQWVmLGFBQWEsQ0FBQztFQUNaLFFBQVEsRUFBRSxRQUFTO0VBQ25CLE9BQU8sRUFBRSxDQUFFLEdBVVo7RUEzQkgsQUFrQkksaUJBbEJhLENBZWYsYUFBYSxDQUdYLElBQUksQ0FBQztJQUNILHlCQUF5QixFQUFFLENBQUU7SUFDN0IsMEJBQTBCLEVBQUUsQ0FBRSxHQU0vQjtJQTFCTCxBQWtCSSxpQkFsQmEsQ0FlZixhQUFhLENBR1gsSUFBSSxBQUdELFNBQVMsRUFyQmhCLEFBa0JJLGlCQWxCYSxDQWVmLGFBQWEsQ0FHWCxJQUFJLEFBR1csTUFBTSxDQUFBO01BQ2pCLEtBQUssRUFBRSxPQUFNO01BQ2IsZ0JBQWdCLEVBQUUsS0FBTTtNQUN4QixZQUFZLEVBQUUsS0FBTSxHQUNyQjs7QUF6QlAsQUE4QkksaUJBOUJhLENBNkJmLHNCQUFzQixDQUNwQixFQUFFLENBQUM7RUFDRCxPQUFPLEVBQUUsZUFBZ0IsR0FDMUI7O0FBaENMLEFBaUNTLGlCQWpDUSxDQTZCZixzQkFBc0IsQ0FJcEIsRUFBRSxDQUFDLENBQUMsQ0FBQyxLQUFLLENBQUM7RUFDVCxTQUFTLEVBQUUsSUFBSztFQUNoQixXQUFXLEVBQUUsTUFBTztFQUNwQixjQUFjLEVBQUUsTUFBTztFQUN2QixXQUFXLEVBQUUsT0FBUSxHQUN0Qjs7QUF0Q0wsQUF5Q0UsaUJBekNlLENBeUNmLEtBQUssQ0FBQztFQUNKLFFBQVEsRUFBRSxRQUFTO0VBQ25CLE9BQU8sRUFBRSxDQUFFO0VBQ1gsT0FBTyxFQUFFLElBQUs7RUFDZCxVQUFVLEVBQUUsT0FBUTtFQUNwQixVQUFVLEVBQUUsR0FBRyxDQUFDLElBQUksQ0FBQyxJQUFJLENBQUMsR0FBRyxDQUFDLHlCQUFJLEVBQXVCLEdBQUcsQ0FBQyxHQUFHLENBQUMsSUFBSSxDQUFDLEdBQUcsQ0FBQyx5QkFBSTtFQUM5RSxNQUFNLEVBQUUsSUFBSyxHQUNkOztBQWhESCxBQWtERSxpQkFsRGUsQ0FrRGYsZ0JBQWdCLENBQUM7RUFDZixVQUFVLEVBQUUsSUFBSztFQUNqQixVQUFVLEVBQUUsS0FBTSxHQUNuQjs7QUFyREgsQUF1REUsaUJBdkRlLENBdURmLEtBQUssQ0FBQztFQUNKLFNBQVMsRUFBRSxJQUFLO0VBQ2hCLFdBQVcsRUFBRSxHQUFJO0VBQ2pCLGNBQWMsRUFBRSxNQUFPO0VBQ3ZCLFdBQVcsRUFBRSxJQUFLLEdBQ25COztBQTVESCxBQThEcUIsaUJBOURKLENBOERmLEtBQUssQ0FBQSxBQUFBLElBQUMsQ0FBRCxRQUFDLEFBQUEsRUFBZTtFQUNuQixNQUFNLEVBQUUsT0FBUTtFQUNoQixLQUFLLEVBQUUsSUFBSztFQUNaLE9BQU8sRUFBRSxZQUFhO0VBQ3RCLFdBQVcsRUFBRSxHQUFJO0VBQ2pCLFlBQVksRUFBRSxHQUFJLEdBQ25COztBQXBFSCxBQXNFRSxpQkF0RWUsQ0FzRWYsV0FBVyxFQXRFYixBQXNFZSxpQkF0RUUsQ0FzRUYsV0FBVyxDQUFDO0VBQ3ZCLE9BQU8sRUFBRSxNQUFPLEdBQ2pCOztBQXhFSCxBQTBFRSxpQkExRWUsQ0EwRWYsV0FBVyxDQUFDO0VBQ1YsS0FBSyxFQUFFLEdBQUk7RUFDWCxZQUFZLEVBQUUsRUFBRyxHQUNsQjs7QUE3RUgsQUErRUUsaUJBL0VlLENBK0VmLFdBQVcsQ0FBQztFQUNWLEtBQUssRUFBRSxHQUFJLEdBQ1o7O0FBakZILEFBbUZFLGlCQW5GZSxDQW1GZixtQkFBbUIsQ0FBQztFQUNsQixPQUFPLEVBQUUsSUFBSztFQUNkLGVBQWUsRUFBRSxhQUFjLEdBQ2hDOztBQXRGSCxBQXlGSSxpQkF6RmEsQ0F3RmYsa0JBQWtCLENBQ2hCLEtBQUssQ0FBQztFQUNKLFNBQVMsRUFBRSxLQUFNLEdBQ2xCOztBQTNGTCxBQTZGTSxpQkE3RlcsQ0F3RmYsa0JBQWtCLENBS2hCLEVBQUUsQUFBQSxZQUFZLENBQUM7RUFDYixVQUFVLEVBQUUsS0FBTTtFQUNsQixPQUFPLEVBQUUsWUFBYSxHQUN2Qjs7QUFoR0wsQUFrR0ksaUJBbEdhLENBd0ZmLGtCQUFrQixDQVVoQixLQUFLLENBQUM7RUFDSixhQUFhLEVBQUUsQ0FBRTtFQUNqQixTQUFTLEVBQUUsSUFBSztFQUNoQixXQUFXLEVBQUUsR0FBSSxHQUNsQjs7QUF0R0wsQUF3RkUsaUJBeEZlLENBd0ZmLDJCQUFrQixDQWdCTDtFQUNULFNBQVMsRUFBRSxJQUFLO0VBQ2hCLFdBQVcsRUFBRSxHQUFJLEdBQ2xCOztBQzNHTCxBQUFBLHdCQUF3QixDQUFDO0VBQ3ZCLFVBQVUsRUFBRSxNQUFPLEdBYXBCO0VBZEQsQUFHRSx3QkFIc0IsQ0FHdEIsS0FBSyxDQUFDO0lBQ0osV0FBVyxFQUFFLElBQUssR0FDbkI7RUFMSCxBQU9FLHdCQVBzQixDQU90QixDQUFDLENBQUM7SUFDQSxLQUFLLEVBQUUsSUFBSyxHQUNiO0VBVEgsQUFXRSx3QkFYc0IsQ0FXdEIsT0FBTyxDQUFDO0lBQ04sV0FBVyxFQUFFLElBQUssR0FDbkI7O0FBSUgsQUFBQSxtQkFBbUIsQ0FBQztFQUNsQixTQUFTLEVBQUUsS0FBTSxHQXFDbEI7RUF0Q0QsQUFFRSxtQkFGaUIsQ0FFakIsS0FBSyxDQUFDO0lBQ0osTUFBTSxFQUFFLE9BQVEsR0FrQ2pCO0lBckNILEFBS0ksbUJBTGUsQ0FFakIsS0FBSyxDQUdILEVBQUUsQ0FBQztNQUNELE9BQU8sRUFBRSxtQkFBb0I7TUFDN0IsVUFBVSxFQUFFLE1BQU8sR0FDcEI7SUFSTCxBQVVJLG1CQVZlLENBRWpCLEtBQUssQ0FRSCxLQUFLLENBQUM7TUFDSixZQUFZLEVBQUUsS0FBTSxHQXlCckI7TUFwQ0wsQUFhUSxtQkFiVyxDQUVqQixLQUFLLENBUUgsS0FBSyxDQUdILEVBQUUsQUFBQSxZQUFZLEVBYnBCLEFBYXdCLG1CQWJMLENBRWpCLEtBQUssQ0FRSCxLQUFLLENBR2EsRUFBRSxBQUFBLFlBQVksQ0FBQztRQUM3QixZQUFZLEVBQUUsSUFBSyxHQUNwQjtNQWZQLEFBaUJNLG1CQWpCYSxDQUVqQixLQUFLLENBUUgsS0FBSyxDQU9ILFlBQVksQ0FBQztRQUNYLEtBQUssRUFBRSxHQUFJLEdBQ1o7TUFuQlAsQUFvQk0sbUJBcEJhLENBRWpCLEtBQUssQ0FRSCxLQUFLLENBVUgsd0JBQXdCLENBQUM7UUFDdkIsS0FBSyxFQUFFLEdBQUksR0FDWjtNQXRCUCxBQXVCTSxtQkF2QmEsQ0FFakIsS0FBSyxDQVFILEtBQUssQ0FhSCxTQUFTLENBQUM7UUFDUixLQUFLLEVBQUUsR0FBSSxHQUNaO01BekJQLEFBMEJNLG1CQTFCYSxDQUVqQixLQUFLLENBUUgsS0FBSyxDQWdCSCxZQUFZLENBQUM7UUFDWCxLQUFLLEVBQUUsR0FBSSxHQUNaO01BNUJQLEFBNkJNLG1CQTdCYSxDQUVqQixLQUFLLENBUUgsS0FBSyxDQW1CSCxlQUFlLENBQUM7UUFDZCxLQUFLLEVBQUUsR0FBSSxHQUNaO01BL0JQLEFBaUNNLG1CQWpDYSxDQUVqQixLQUFLLENBUUgsS0FBSyxDQXVCSCxZQUFZLENBQUM7UUFDWCxLQUFLLEVBQUUsR0FBSSxHQUNaOztBQUtQLEFBQUEsU0FBUyxDQUFDO0VBQ1IsU0FBUyxFQUFFLEtBQU0sR0FDbEI7O0FBRUQsQUFBVSxTQUFELENBQUMsSUFBSSxDQUFDO0VBQ2IsT0FBTyxFQUFFLFlBQWE7RUFDdEIsWUFBWSxFQUFFLEdBQUksR0FDbkIiLCJuYW1lcyI6W119 */
`

I am not sure where is this coming from!

Reply:
Nice attention to detail! I don't have a strong/informed opinion---this is definitely legacy build system. My vote is we leave it as the gulpfile generated, test it in a staging environment, and worst case release a patch that removes it.